README.md

```diff
@@ -23,50 +25,144 @@ You will need to create an account at https://tempestwx.com/ and then generate a
+` ` | solar_radiation | W/m^2 | W/m^2 | occupancy_trigger_value | 1000| 1000 |
+` ` | uv | Index | Index | occupancy_trigger_value | 3 | 3 |
+` ` | wind_direction | degrees | degrees | occupancy_trigger_value | 360 | 360 |
+
+`{*}`  Reference: https://weatherflow.github.io/Tempest/api/swagger/#!/observations/getStationObservation
+
+`{**}` Reference Wiki for details on how to view Occupancy Sensor values with iOS 16.x and MacOS Ventura 13.x.
 
-#### Config Example
+### Config Example
 
 ```json
 {
   "name": "WeatherFlow Tempest Platform",
   "token": "XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX",
   "station_id": 10000,
   "interval": 10,
+  "units": "Standard",
   "sensors": [
-    {
-      "name": "Outside Temperature",
-      "sensor_type": "Temperature Sensor",
-      "value_key": "air_temperature"
-    },
-    {
-      "name": "Outside Relative Humidity",
-      "sensor_type": "Humidity Sensor",
-      "value_key": "relative_humidity"
-    },
-    {
-      "name": "Outside Light Level",
-      "sensor_type": "Light Sensor",
-      "value_key": "brightness"
-    },
-    {
-      "name": "Outside Wind Speed",
-      "sensor_type": "Fan",
-      "value_key": "wind_avg"
-    },
-    {
-      "name": "Wind Speed Detector",
-      "sensor_type": "Motion Sensor",
-      "value_key": "wind_gust",
-      "additional_properties": {
-        "motion_trigger_value": 30
+      {
+          "name": "Temperature",
+          "sensor_type": "Temperature Sensor",
+          "temperature_properties": {
+              "value_key": "air_temperature"
+          }
+      },
+      {
+          "name": "Feels Like",
+          "sensor_type": "Temperature Sensor",
+          "temperature_properties": {
+              "value_key": "feels_like"
+          }
+      },
+      {
+          "name": "Dew Point",
+          "sensor_type": "Temperature Sensor",
+          "temperature_properties": {
+              "value_key": "dew_point"
+          }
+      },
+      {
+          "name": "Relative Humidity",
+          "sensor_type": "Humidity Sensor",
+          "humidity_properties": {
+              "value_key": "relative_humidity"
+          }
+      },
+      {
+          "name": "Light Level",
+          "sensor_type": "Light Sensor",
+          "light_properties": {
+              "value_key": "brightness"
+          }
+      },
+      {
+          "name": "Wind Speed",
+          "sensor_type": "Fan",
+          "fan_properties": {
+              "value_key": "wind_avg"
+          }
+      },
+      {
+          "name": "Wind Gust",
+          "sensor_type": "Motion Sensor",
+          "motion_properties": {
+              "value_key": "wind_gust",
+              "motion_trigger_value": 10
+          }
+      },
+      {
+          "name": "Barometer",
+          "sensor_type": "Occupancy Sensor",
+          "occupancy_properties": {
+              "value_key": "barometric_pressure",
+              "occupancy_trigger_value": 30
```

Comment:
Because the top-level key is `occupancy_properties ` I think it's redundant to say `occupancy_trigger_value` it could simply be `trigger_value`.

Reply:
Already incorporated in v2.1.9 which will be carried forward into v3.0.0.